services/tenant-ui/frontend/src/components/issuance/credentialDefinitions/CreateCredentialDefinition.vue

```diff
@@ -46,13 +48,89 @@ defineEmits(['success']);
 
 const { t } = useI18n();
 
-const { isIssuer } = storeToRefs(useTenantStore());
+const { isIssuer, tenantWallet } = storeToRefs(useTenantStore());
 const { schemaList, selectedCredentialDefinition } =
   storeToRefs(useGovernanceStore());
 const governanceStore = useGovernanceStore();
 
 const formattedSchemaList = computed(() => formatSchemaList(schemaList));
 
+// Check if wallet is askar-anoncreds
+const isAskarAnoncredsWallet = computed(() => {
+  if (!tenantWallet) return false;
+  return tenantWallet.value?.settings?.['wallet.type'] === 'askar-anoncreds';
+});
+
+// WebVH configuration state
+const webvhConfig = ref<any>(null);
+const webvhConfigLoaded = ref(false);
+
+// Check if webvh endorser is connected
+const isWebvhEndorserConnected = computed(() => {
+  if (!isAskarAnoncredsWallet.value) {
+    return true; // For non-askar-anoncreds wallets, always allow
+  }
+
+  if (!webvhConfig.value) {
+    return false;
+  }
+
+  const witnesses = webvhConfig.value.witnesses;
+  const isConnected = Boolean(
+    witnesses && Array.isArray(witnesses) && witnesses.length > 0
+  );
+
+  return isConnected;
+});
+
+// Disable button logic:
+// - For askar wallets: require isIssuer to be true
+// - For askar-anoncreds wallets: only require WebVH endorser to be connected
+const isCreateButtonDisabled = computed(() => {
+  if (isAskarAnoncredsWallet.value) {
+    return !isWebvhEndorserConnected.value;
+  } else {
+    return !isIssuer?.value;
+  }
+});
```

Comment:
Lets not reproduce this stuff per component, it can live in a store, though see comment above about wrapping what we can into isIssuer.